src/components/Common/Header.jsx

```diff
@@ -0,0 +1,29 @@
+
+import React from 'react';
+
+export default class Header extends React.Component {
+
+  handleChange(e) {
+    const tb_title = e.target.value;
+    this.props.topLevelModule1(tb_title);
+  }
+
+  render() {
+    if (this.props.txtBoxVisible) {
+    return (
+      <div>
+          <header><h1>{this.props.txtBoxVisible}</h1></header>
+          <header><h1>{this.props.title}</h1></header>
+          <input value={this.props.title} onChange={this.handleChange.bind(this)}/>
```

Comment:
I was using the training page to try out a couple of things.  The text box copies back to the header text immediately.  It was to see if I could introduce an input box in the props data flow.  I will strip it out.